async-request-reply/README.md

```diff
@@ -75,30 +101,77 @@ For more information about this pattern, see [Asynchronous Request-Reply pattern
 
    > **Note** the app uses the WEBSITE_HOSTNAME environment variable. This environment variable is set automatically by the Azure App Service runtime environment. For more information, see [Azure runtime environment](https.://github.com/projectkudu/kudu/wiki/Azure-runtime-environment)
 
-### Clean up
+## :broom: Clean up resources
 
-1. Delete all the Azure resources for this Cloud Async Pattern
+Most of the Azure resources deployed in the prior steps will incur ongoing charges unless removed.
 
    ```bash
-   az group delete -n rg-asyncrequestreply -y
+   az group delete -n ${RESOURCEGROUP} -y
    ```
 
-### Running localy
+## Running locally
 
 You could open the solution with Visual Studio, then you need to create on the root `local.settings.json`
 
-   ```json
-    {
-    "IsEncrypted": false,
-    "Values": {
-        "AzureWebJobsStorage": "UseDevelopmentStorage=true",
-        "FUNCTIONS_WORKER_RUNTIME": "dotnet",
-        "ServiceBusConnectionAppSetting": "<yourdata>",
-        "StorageConnectionAppSetting": "<yourData>"
-    }
-    }
-   ```
\ No newline at end of file
+```json
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated",
+    "ServiceBusConnection__fullyQualifiedNamespace": "<yourData>",
+    "DataStorage__blobServiceUri": "<yourData>"
+  }
+}
+```
+
+As far the implementation is using manage identity, you need to assign the role to your [developer identity](https://learn.microsoft.com/azure/azure-functions/functions-reference?tabs=blob&pivots=programming-language-csharp#local-development-with-identity-based-connections).
+
+You need to add the following lines to the Bicep file to assign roles for Service Bus and Azure Storage permissions.
+
+```bicep
+// Assign Role to allow sending messages to the Service Bus
+resource serviceBusSenderRoleAssignmentUser 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
+  name: guid(resourceGroup().id, 'LocalUser', 'ServiceBusSenderRole')
+  scope: serviceBusNamespace
+  properties: {
+    roleDefinitionId: senderServiceBusRole
+    principalId: <your user object id>
+    principalType: 'User'
+  }
+}
+
+// Assign Role to allow receiving messages from the Service Bus
+resource serviceBusReceiverRoleAssignmentUser 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
+  name: guid(resourceGroup().id, 'LocalUser', 'ServiceBusReceiverRole')
+  scope: serviceBusNamespace
+  properties: {
+    roleDefinitionId: receiverServiceBusRole
+    principalId: <your user object id>
+    principalType: 'User'
+  }
+}
+
+// Assign Role to allow Read, write, and delete Azure Storage containers and blobs. 
+resource dataStorageBlobDataContributorRoleAssignmentUser 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
+  name: guid(resourceGroup().id, 'LocalUser', 'StorageBlobDataContributorRole')
+  scope: dataStorageAccount
+  properties: {
+    roleDefinitionId: receiverServiceBusRole
```

Comment:
`receiverServiceBusRole` is used here, but this says it's for storage?  Was this tested?

Reply:
It was a mistake creating the readme. These are the same roles that we give to the funtion apps, but to the user to run locally.  I fix it.